flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -160,10 +169,10 @@ public static NetworkEnvironment create(
 			singleInputGateFactory);
 	}
 
-	private static void registerNetworkMetrics(MetricGroup metricGroup, NetworkBufferPool networkBufferPool) {
+	private static void registerNetworkMetrics(String groupName, MetricGroup metricGroup, NetworkBufferPool networkBufferPool) {
```

Comment:
It might be better to put this method together with other metric related methods below, avoid dispersed anywhere.

Reply:
I would avoid history change in this commit, could be a followup to put all metric methods into a `NettyShuffleMetricFactory` with constants.